src/workers/AGENTS.md

```diff
@@ -69,15 +71,50 @@ import { MatchedCandidate } from '@dailydotdev/schema';
 
 const worker: TypedWorker<'gondul.v1.candidate-opportunity-match'> = {
   subscription: 'api.store-candidate-opportunity-match',
-  handler: async ({ data }, con): Promise<void> => {
-    // data is typed as MatchedCandidate
+  handler: async (message, con): Promise<void> => {
+    // message.data is typed as MatchedCandidate
+    const { data } = message;
   },
   parseMessage: (message) => {
```

Comment:
we need to say that adding `parseMessage` is required if `data` is protobuf or from `@dailydotdev/schema`, linter also warns

Reply:
The note above already mentions it. but I'll push a clarification in couple places